src/components/UniversalTable/UniversalTable.module.scss

```diff
@@ -41,12 +41,16 @@
   }
   
   &Cell {
-    text-align: start;
+    &:nth-child(1) {
```

Comment:
why are yo so sure it won't brake anything on other pages that use the same component?

Reply:
Each content in first column shoud aligned to start of cell.